pkg/network/tracer/tracer_test.go

```diff
@@ -66,133 +65,6 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestTracerExpvar(t *testing.T) {
```

Comment:
We could keep this test and call the `GetStats` function instead, right?

Reply:
sure, I can add this back 👍